client/me/purchases/remove-purchase/index.jsx

```diff
@@ -117,12 +118,18 @@ class RemovePurchase extends Component {
 	};
 
 	changeSurveyStep = stepFunction => {
-		const { selectedPurchase, isChatAvailable, isChatActive } = this.props;
+		const {
+			selectedPurchase,
+			isChatAvailable,
+			isChatActive,
+			precancellationChatAvailable = false,
```

Comment:
Since this is a "connected property" from redux state, I personally prefer to let the selector handling this. See the inlined comment below for more details :)